packages/ui/src/ui-component/TooltipWithParser.js

```diff
@@ -0,0 +1,13 @@
+import { Info } from '@mui/icons-material';
+import { IconButton, Tooltip } from '@mui/material';
+import parser from 'html-react-parser';
+
+export const TooltipWithParser = ({ title }) => {
+    return (
+        <Tooltip title={parser(title)} placement="right">
+            <IconButton>
+                <Info style={{ height: 18, width: 18 }} />
+            </IconButton>
+        </Tooltip>
+    );
+};
```

Comment:
Warning `'title' is missing in props validation  react/prop-types`
Need to `import PropTypes from 'prop-types';` and use it
`

Reply:
resolved